draft-ietf-httpbis-digest-headers.md

```diff
@@ -243,19 +246,21 @@ dependent on other transformations (e.g. transfer codings for HTTP/1.1 - see
 several examples are provided in {{resource-representation}}.
 
 When a message has no representation data it is still possible to assert that no
-representation data was sent computing the representation digest on an empty
+representation data was sent by computing the digest on an empty
 string (see {{usage-in-signatures}}).
 
-`Representation-Digest` is a Structured Fields Dictionary (see {{Section 3.2 of
-STRUCTURED-FIELDS}}):
+`Representation-Digest` is a Structured Fields `Dictionary` (see {{Section 3.2 of
+STRUCTURED-FIELDS}}) where:
+* members cannot have parameters;
```

Comment:
This is something I sidestepped during the initial change to SF. The SF grammar allows parameters, so member-values *can* have them. The SF says

```
   Both Items and Inner Lists allow parameters as an extensibility
   mechanism; this means that values can later be extended to
   accommodate more information, if need be.  To preserve forward
   compatibility, field specifications are discouraged from defining the
   presence of an unrecognized parameter as an error condition.
```

When updating Digest we decided parameters could be a security problem. If we want prohibit the use of SF parameters, then we need some form of normative text to state that and probably justify why. I'd suggest doing that in a separate paragraph.

Reply:
Furthermore, if we move the parameter discussion elsewhere, then we can get rid of the list format and use a prose style similar to what's already there.

Reply:
@LPardue I added a paragraph in the security consideration.